src/plays/drawing-board/index.css

```diff
@@ -0,0 +1,52 @@
+@import "../../../node_modules/xp.css/dist/XP.css";
+
+canvas {
+  transform: translate3d(-50%, 0, 0);
+  cursor: url(./assets/pencil.png) 0 34, auto;
+  margin: 100px 50%;
+}
+
+.colors {
```

Comment:
Please add a unique class name as a prefix to all your styles. Otherwise, it may impact other styles. I see some here already(the like button)

![image](https://user-images.githubusercontent.com/3633137/202613799-f2329b57-6aa1-43d2-b530-ee0e2aa19fe3.png)